### PR TITLE
Use `_choice` suffix consistently in Protobuf

### PR DESCRIPTION
**Pull request by TomGneuss**
In #531 the `_choice` suffix has been introduced for concrete types with concrete classes as subtypes. Since Protobuf does not support inheritance, this was necessary so that we can distinguish between the concrete class and a choice (union) class with quasi-polymorphism. For the sake of consistency, this suffix shall be used for all types that have concrete descendants, *i.e.* for all the choice (union) classes, including the abstract classes.

### CODE REVIEW

**Comment by TomGneuss** [comment]
Apparently, commits from an old PR of mine are still included. Sorry for the mess. Only the latest 2 commits are relevant to this PR. Refer to the list of changed files to confirm this.

File: aas_core_codegen/protobuf/common.py [hunk 1]
```diff
@@ -139,14 +139,12 @@ def generate_type(type_annotation: intermediate.TypeAnnotationUnion) -> Stripped
...
-                isinstance(our_type, intermediate.ConcreteClass)
-                and len(our_type.concrete_descendants) > 0
-            ):
-                # NOTE (mristin):
-                # We have to add the suffix ``_choice`` since this field points
-                # to one of the concrete descendants of the class as well as
-                # the concrete class itself.
+            if len(our_type.concrete_descendants) > 0:
+                # NOTE (TomGneuss):
+                # add the suffix "_choice" to the type name because this type
```
Comment (mristin) [inline aas_core_codegen/protobuf/common.py:144]:
```suggestion
                # We add the suffix "_choice" to the type name because this type
```

**Review by mristin — APPROVED** [review]
LGTM! I made minor cosmetic changes.

**Comment by mristin** [comment]
Thanks, @TomGneuss !

**Pull request merged**